.changeset/new-comics-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/platform-cli': minor
+---
+
+Rely on eslint-config-next's parser instead of the deprecated babel-eslint parser
```

Comment:
Want to add another bit here about the `fix` option change?

Reply:
Yes, good call!